packages/product/src/services/product.ts

```diff
@@ -72,17 +72,17 @@ export default class ProductService<TEntity extends Product = Product> {
     config: FindConfig<ProductTypes.ProductDTO> = {},
```

Comment:
thought(this-file): This is a breaking change no? Should be a minor bump. If you agree, I can add it in a separate PR

Reply:
Yes, agree! 🎉